src/SFA.DAS.AANHub.Api/Controllers/ApprenticesController.cs

```diff
@@ -63,6 +66,30 @@ public async Task<ActionResult<GetApprenticeMemberResult>> GetApprentice(long ap
             return new OkObjectResult(apprenticeMemberResult);
 
         }
-    }
-}
 
+        /// <summary>
+        /// Patch an apprentice member
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <param name="apprenticeid"></param>/// 
```

Comment:
`apprenticeid` should be `apprenticeId`

Reply:
`apprenticeid` has now been updated to `apprenticeId`